libr/core/cmd_open.c

```diff
@@ -236,8 +236,7 @@ static void cmd_open_map (RCore *core, const char *input) {
 	}
 	r_core_block_read (core);
 }
-
-R_API void r_core_file_reopen_in_malloc(RCore *core) {
+R_API void r_core_file_reopen_in_malloc (RCore *core) {
```

Comment:
im talking about the first commit where some code was moved to file.c this is the reason of the second commit leaving this files untouched